includes/wikia/tests/UrlHelpersTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+class UrlHelpersTest extends WikiaBaseTest {
+
+	/**
+	 * @dataProvider normalizeHostDataProvider
+	 */
+	public function testNormalizeHost( $environment, $host, $expected ) {
+		$this->mockEnvironment( $environment );
+		$url = wfNormalizeHost( $host );
+		$this->assertEquals( $expected, $url );
+	}
+
+	public function normalizeHostDataProvider() {
+		return [
+			[
+				'env' => WIKIA_ENV_DEV,
+				'url' => 'http://muppet.' . static::MOCK_DEV_NAME . '.wikia-dev.us',
+				'expected' => 'http://muppet.wikia.com'
+			],
+			[
+				'env' => WIKIA_ENV_DEV,
+				'url' => 'http://muppet.' . static::MOCK_DEV_NAME . '.fandom-dev.us',
+				'expected' => 'http://muppet.fandom.com'
```

Comment:
There is no http without s in fandom.com :lock:

Reply:
👍 https://github.com/Wikia/app/pull/16068/commits/f4985582c5c63580bd3ca401f79e5fc7c5758949